website_event_waiting_list/templates/waiting_list_registration.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template name="Event waiting list registration" id="event_waiting_list_registration">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="oe_structure"/>
+                <div class="container">
+                    <h1><span t-field="values.get('event').name" /> Waiting list</h1>
+                    <div class="row">
+                        <div class="col-md-8">
+                            <div class="oe_structure">
+                                <div>
+                                    <p>Register to this event's waiting list.</p>
+                                    <p>We'll do our best to get back to you as soon as possible.</p>
+                                </div>
+                            </div>
+                            <form action="/website_form/"
+                                  method="post"
+                                  data-model_name="crm.lead"
+                                  t-att-data-success_page="'%s/waiting-list-confirmation' % values.get('event').website_url"
```

Comment:
event.website_url

Reply:
no need to access 'values' -> rendered values are already in the scope w/ their own names